drivers/timers/pwm.c

```diff
@@ -602,7 +603,7 @@ int pwm_register(FAR const char *path, FAR struct pwm_lowerhalf_s *dev)
    * kmm_zalloc()).
    */
 
-  nxsem_init(&upper->exclsem, 0, 1);
+  nxmutex_init(&upper->lock);
```

Comment:
ditto. either keep like it is or add error handling if `register_driver` fails

Reply:
keep